index.d.ts

```diff
@@ -0,0 +1,41 @@
+interface AbortError extends Error {}
+
+interface AbortErrorConstructor {
+	/**
+	 * Abort pending execution. All unresolved promised are rejected with a `AbortError` error.
+	 *
+	 * @param message - A human-readable description of the error.
+	 */
+    new(message?: string): AbortError;
+	/**
+	 * Abort pending execution. All unresolved promised are rejected with a `AbortError` error.
+	 *
+	 * @param message - A human-readable description of the error.
+	 */
+	(message?: string): AbortError;
+    readonly prototype: AbortError;
+}
+
+export const AbortError: AbortErrorConstructor;
+
+/**
+ * Returns a throttled version of `fn`.
+ *
+ * @param fn - Promise-returning/async function or a normal function.
```

Comment:
Do you think we should use a better name than `fn` here? Is it allowed to use `function` in TS? Otherwise, maybe just `input` since it accepts both a function and async function.

Reply:
Function is not possible. Went with `input`.